src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteProperties.DateTime.cs

```diff
@@ -0,0 +1,278 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Buffers.Text;
+using System.Diagnostics;
+
+namespace System.Text.Json
+{
+    public ref partial struct Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the property name and <see cref="DateTime"/> value (as a JSON string) as part of a name/value pair of a JSON object.
+        /// </summary>
+        /// <param name="propertyName">The UTF-16 encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        /// <param name="value">The value to be written as a JSON string as part of the name/value pair.</param>
+        /// <param name="suppressEscaping">If this is set, the writer assumes the property name is properly escaped and skips the escaping step.</param>
+        /// <exception cref="ArgumentException">
+        /// Thrown when the specified property name is too large.
+        /// </exception>
+        /// <exception cref="JsonWriterException">
+        /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
+        /// </exception>
+        public void WriteString(string propertyName, DateTime value, bool suppressEscaping = false)
+            => WriteString(propertyName.AsSpan(), value, suppressEscaping);
+
+        /// <summary>
+        /// Writes the property name and <see cref="DateTime"/> value (as a JSON string) as part of a name/value pair of a JSON object.
+        /// </summary>
+        /// <param name="propertyName">The UTF-16 encoded property name of the JSON object to be transcoded and written as UTF-8.</param>
+        /// <param name="value">The value to be written as a JSON string as part of the name/value pair.</param>
+        /// <param name="suppressEscaping">If this is set, the writer assumes the property name is properly escaped and skips the escaping step.</param>
+        /// <exception cref="ArgumentException">
+        /// Thrown when the specified property name is too large.
+        /// </exception>
+        /// <exception cref="JsonWriterException">
+        /// Thrown if this would result in an invalid JSON to be written (while validation is enabled).
+        /// </exception>
+        public void WriteString(ReadOnlySpan<char> propertyName, DateTime value, bool suppressEscaping = false)
+        {
+            JsonWriterHelper.ValidateProperty(ref propertyName);
+
+            if (!suppressEscaping)
```

Comment:
Inconsistent curly braces

Reply:
I don't think this violates the coding guidelines (since we are the method scope). I am leaning towards removing the braces in other areas rather (for consistency) than adding them here. For example, removing them from: https://github.com/dotnet/corefx/pull/34425/files/9bec4782491ea4d782d0385956de723930e8b5a6#diff-62946db0e47e821e4f0c51eb8bcdf1f3R84

Reply:
> I, of course, would rather they be added :) (The lack of braces in my personal style are only acceptable in the if/throw preamble)

Should we, then, update the coding style guideline accordingly?